<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WdgColorOptions</class>
 <widget class="QWidget" name="WdgColorOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>663</width>
    <height>269</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>255</horstretch>
    <verstretch>255</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>425</width>
    <height>200</height>
   </size>
  </property>
  <widget class="QCheckBox" name="m_randomOpacity">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Random opacity</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>381</width>
     <height>94</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QCheckBox" name="hueChBox">
      <property name="text">
       <string>Hue</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSlider" name="hueSlider">
      <property name="minimum">
       <number>-180</number>
      </property>
      <property name="maximum">
       <number>180</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksAbove</enum>
      </property>
      <property name="tickInterval">
       <number>20</number>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QSpinBox" name="hueBox">
      <property name="minimum">
       <number>-180</number>
      </property>
      <property name="maximum">
       <number>180</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QCheckBox" name="saturationChBox">
      <property name="text">
       <string>Saturation</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSlider" name="saturationSlider">
      <property name="minimum">
       <number>-100</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksAbove</enum>
      </property>
      <property name="tickInterval">
       <number>11</number>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QSpinBox" name="saturationBox">
      <property name="minimum">
       <number>-100</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QCheckBox" name="valueChBox">
      <property name="text">
       <string>Value</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSlider" name="valueSlider">
      <property name="minimum">
       <number>-100</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksAbove</enum>
      </property>
      <property name="tickInterval">
       <number>11</number>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QSpinBox" name="valueBox">
      <property name="minimum">
       <number>-100</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>m_randomOpacity</tabstop>
  <tabstop>hueSlider</tabstop>
  <tabstop>saturationSlider</tabstop>
  <tabstop>valueSlider</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>hueSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hueBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>376</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hueBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hueSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>381</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>298</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saturationSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>saturationBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>377</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saturationBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>saturationSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>389</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>valueSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>valueBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>298</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>370</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>valueBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>valueSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>376</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>302</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
